pkg/kudoctl/util/repo/repo_operator.go

```diff
@@ -79,21 +79,43 @@ func (r *Client) DownloadIndexFile() (*IndexFile, error) {
 }
 
 // getPackageReaderByFullPackageName downloads the tgz file from the remote repository and unmarshals it to the package CRDs
```

Comment:
also this part of comment is not right, right? I don't see any unmarshalling to CRDs 🤔